bin/win/install.bat

```diff
@@ -0,0 +1,5 @@
+REM Installer for DMIS on Win,
```

Comment:
might just leave this as a shortcut when setting up the venv for running locally